### ISSUE TITLE
Color scheme in sidebar is very difficult to read

### ISSUE THREAD

**Issue by robdaemon** [AdaCore/learn#828]
On the table of contents sidebar, the orange text on varying shades of gray is difficult to read. For accessibility reasons, colors with more contrast should be chosen (or a separate stylesheet as an option?)

![e6bef6dda306801f](https://user-images.githubusercontent.com/287213/210156882-53f0a9ac-8b41-4f75-98f9-e9eacc7838bf.jpeg)

This is a photo taken with an iPhone 13 Pro of the text on a "2k" Chromebook screen.
